upcoming_changes/162.new.rst

```diff
@@ -0,0 +1,5 @@
+Added the :func:`~rsciio.utils.distributed.memmap_distributed` function for loading a memmap file
+from multiple processes.
+
+Added the arguments ``distributed`` and ``metadata_file`` to the .mrc file reader for loading metadata
+save from DirectElectron detectors.
```

Comment:
Is it worth mentioning the speed improvement (remove reshape, transpose) from the last commit?

Reply:
Done!